content/en/security/cloud_security_management/setup/without_infrastructure_monitoring.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Setting Up CSM without Infrastructure Monitoring
+---
+
+You can set up CSM without infrastructure monitoring, so you can save on costs.
+
+**Tip**: In your CSM settings, set up [resource evaluation filters][1] to limit the number of hosts you need security on.
+
+## Set up CSM on your AWS account
+
+1. Go to the [AWS Integration configuration page][2] in Datadog.
+1. On the **Configuration** tab, select the account you want to enable CSM on.
+1. To turn off infrastructure monitoring, under the selected account number, on the **Metric Collection** tab, click the **disable metric collection** link. Then, click **Disable Metric Collection** to confirm.
```

Comment:
1. To turn off infrastructure monitoring on a given account, under the selected account number, on the **Metric Collection** tab, click the **disable metric collection** link. Then, click **Disable Metric Collection** to confirm.